voicer-client/src/components/voice/Voice.jsx

```diff
@@ -53,18 +53,11 @@ export default function Voice() {
   }, [searchTags]);
 
   useEffect(()=> {
+    console.log(voiceSearch.split(' '))
     if(voiceSearch === ''){
       setSearchTags("")
     }else{
-      setSearchTags(voiceSearch.split(' ').map((tag, index) => {
-        if(index===0){
-          console.log(index)
-          return `?tag${index}=${tag}`
-        }else{
-          console.log(index)
-          return `&tag${index}=${tag}`
-        }
-      }).join(''))
+      setSearchTags(`?tag=${voiceSearch.split(' ')}`)
```

Comment:
9 lines of code --> 1: love it! Nice work.

Reply:
thanks!